src/docs/user/references/VariablesReference.adoc

```diff
@@ -34,7 +34,7 @@ The order in this list corresponds to the task life-cycle order.
 
 | *Task arguments*. Arguments of a script defined in the workflow. Available via all kind of scripts, when referencing an external file or directly as XML elements.
 | ... = args[0];
-| Passed to native executable.
+| Passed to native executable. Can also be used with $args_0
```

Comment:
'Can be also used with' instead of 'Can also be used with'

Reply:
both correct anyway I think

Reply:
Actually "Can also be used" is more correct